FileConv/Gfx/PrintShop-notes.md

```diff
@@ -39,13 +42,42 @@ the cyan plane.
 The actual color palette, as determined by a screen grab from an emulator, is comprised of RGB
 primaries and purple/orange that are similar to the classic Apple II hi-res colors.
 
-Y M C | color  | red, green, blue
------ | ------ | ----------------
-0 0 0 | white  | 0xff, 0xff, 0xff
-0 0 1 | blue   | 0x00, 0x00, 0xff
-0 1 0 | red    | 0xff, 0x00, 0x00
-0 1 1 | purple | 0xcc, 0x00, 0xcc
-1 0 0 | yellow | 0xff, 0xff, 0x00
-1 0 1 | green  | 0x00, 0xff, 0x00
-1 1 0 | orange | 0xff, 0x66, 0x00
-1 1 1 | black  | 0x00, 0x00, 0x00
+| Y M C | color  | red, green, blue |
+| ----- | ------ | ---------------- |
+| 0 0 0 | white  | 0xff, 0xff, 0xff |
+| 0 0 1 | blue   | 0x00, 0x00, 0xff |
+| 0 1 0 | red    | 0xff, 0x00, 0x00 |
+| 0 1 1 | purple | 0xcc, 0x00, 0xcc |
+| 1 0 0 | yellow | 0xff, 0xff, 0x00 |
+| 1 0 1 | green  | 0x00, 0xff, 0x00 |
+| 1 1 0 | orange | 0xff, 0x66, 0x00 |
+| 1 1 1 | black  | 0x00, 0x00, 0x00 |
+
+## Fonts
+
+Each font contains 59 characters (0-58).
+
+In the Print SHop Companion (PSC) font editor, offsets 0 (space) and 32 (image) are not editable.
+
+In Print Shop[, offset 32 points to the current graphic selected, which
+is generally not contained in the font file itself.  However, since it is an arbitrary data pointer, it is
+conceivble that there could be self-contained data in the file.
+
+The Space character is at offset 0, and in all the examples I've seen, has a height of 0 pixels.
+The data is calculated in the PSC font editor based off of an average of all other character widths.
+
+$5FF4-$5FFF - Optional Print Shop Companion Font Header (From Font Editor)
+$6000-$603A - Character widths (in pixels).  High bit indicates "Edited" in PSC.  Should be stripped.
+$603B-$6075 - Character heights in rows.
+$6076-$60B0 - Low Bytes of character data pointers
+$60B1-$60EB - High Bytes of character data pointers
+$60EC-EOF   - Character data.
```

Comment:
Can you briefly describe the character data?  One bit per pixel, 0=background / 1=foreground, LSB first, bytes left to right, series of rows, no padding between characters, ...?  The code is the One True Description, but a few words here would be helpful.

Reply:
Done.